extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -2101,6 +2108,27 @@ export class AmpA4A extends AMP.BaseElement {
   getIframeTitle() {
     return this.element.getAttribute('title') || '3rd party ad content';
   }
+
+  /**
+   * Queries the dom through the `runtime-type` meta. If the value is one of
+   * 2, 4, 10 (which corresponds to module/nomodule or its control diversion)
+   * and returns the correct experiment ID.
+   *
+   * @protected
+   * @return {?string}
+   */
+  getModuleNomoduleExpIds_() {
+    const runtimeType = this.getAmpDoc().getMetaByName('runtime-type');
+    // ModuleNomoduleControl = 10 (current default, just nomodule)
+    if (runtimeType === '10') {
+      return MODULE_NOMODULE_PARAMS_EXP.CONTROL;
+    }
+    // ES6 = 2, ES6 No Preload = 4 (module/nomodule mode)
+    if (runtimeType === '2' || runtimeType === '4') {
+      return MODULE_NOMODULE_PARAMS_EXP.EXPERIMENT;
```

Comment:
Are you planning to run both two and four simultaneously? If so, perhaps you should have two different experiment ids to represent these two different branches?

Reply:
actually we never run 4. i'll go ahead and remove it